cogs/obliterate.py

```diff
@@ -334,6 +397,17 @@ async def apply(self, interaction: discord.Interaction):
             return
         await interaction.response.send_modal(AccountInfoModal(self.bot))
 
+    @app_commands.command()
+    @app_commands.guilds(discord.Object(id=config['obliterate_guild_id']), discord.Object(id=config['test_guild_id']))
+    async def appreciate(self, *appreciate_players, interaction: discord.Interaction):
+        '''
+        Send a modal with the appreciation station form.
+        '''
+        appreciation_station_channel = interaction.guild.get_channel(config['obliterate_appreciation_station_channel_id'])
```

Comment:
Reminder: before merging, create an appreciation channel and add its id to `data/config.json`

Reply:
I don't have access to the config file as the `data` folder is added to your `.gitignore` :) So you'll need to add that once we have a channel 🙏

Reply:
Yes, that is the intention! This reminder was meant for myself 😜

Reply:
Ahhhh ok 😄